WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -95,14 +97,23 @@ class NoticePresenter: NSObject {
 
         animatePresentation(fromState: fromState, toState: toState, completion: {
             // Quick Start notices don't get automatically dismissed
-            guard notice.style != .quickStart else {
+            if case .quickStart(_) = notice.style {
                 return
             }
 
             DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + Animations.dismissDelay, execute: dismiss)
         })
     }
 
+    private func getNoticeView(for notice: Notice) -> NoticePresentationView {
```

Comment:
Maybe `makeNoticeView` to be consistent with my earlier comment?

Reply:
Sounds great.